<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>card_editor_main</class>
 <widget class="QMainWindow" name="card_editor_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>T.I.M.E Stories Card Editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="card_editor_res.qrc">
    <normaloff>:/images/icon</normaloff>:/images/icon</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,10">
    <item>
     <widget class="QLabel" name="lblInfo">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="wCardView" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>893</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuCard_zoom">
     <property name="title">
      <string>Card zoom</string>
     </property>
     <addaction name="actionZoomIn"/>
     <addaction name="actionZoomReset"/>
     <addaction name="actionZoomOut"/>
    </widget>
    <addaction name="actionFrontFace"/>
    <addaction name="menuCard_zoom"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dwCards">
   <property name="windowTitle">
    <string>Cards</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTreeWidget" name="twCards">
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::InternalMove</enum>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="rootIsDecorated">
         <bool>true</bool>
        </property>
        <property name="uniformRowHeights">
         <bool>true</bool>
        </property>
        <property name="columnCount">
         <number>2</number>
        </property>
        <attribute name="headerVisible">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Info</string>
         </property>
        </column>
       </widget>
       <widget class="QStackedWidget" name="swCardProps">
        <widget class="QWidget" name="pgunknown"/>
        <widget class="QWidget" name="page_4"/>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dwAssets">
   <property name="windowTitle">
    <string>Assets</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTreeWidget" name="twAssets">
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::InternalMove</enum>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="rootIsDecorated">
         <bool>true</bool>
        </property>
        <property name="uniformRowHeights">
         <bool>true</bool>
        </property>
        <property name="columnCount">
         <number>1</number>
        </property>
        <attribute name="headerVisible">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">Name</string>
         </property>
        </column>
       </widget>
       <widget class="QGraphicsView" name="gvAsset"/>
       <widget class="QStackedWidget" name="swAssetProps">
        <widget class="QWidget" name="pgnone"/>
        <widget class="QWidget" name="pgfile">
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLabel" name="lblFilename">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblSize">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>17</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pgimage"/>
        <widget class="QWidget" name="pgstyle"/>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New deck</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load deck...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save deck...</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save deck as...</string>
   </property>
  </action>
  <action name="actionFrontFace">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Front face</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoomReset">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Reset zoom</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="action100">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>100%</string>
   </property>
  </action>
  <action name="action125">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>125%</string>
   </property>
  </action>
  <action name="action150">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>150%</string>
   </property>
  </action>
  <action name="action175">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>175%</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="card_editor_res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>card_editor_main</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>card_editor_main</receiver>
   <slot>do_about()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLoad</sender>
   <signal>triggered()</signal>
   <receiver>card_editor_main</receiver>
   <slot>do_load()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNew</sender>
   <signal>triggered()</signal>
   <receiver>card_editor_main</receiver>
   <slot>do_new()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>card_editor_main</receiver>
   <slot>do_save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSaveAs</sender>
   <signal>triggered()</signal>
   <receiver>card_editor_main</receiver>
   <slot>do_saveas()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>twAssets</sender>
   <signal>currentItemChanged(QTreeWidgetItem*,QTreeWidgetItem*)</signal>
   <receiver>card_editor_main</receiver>
   <slot>current_asset_changed(QTreeWidgetItem*,QTreeWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>883</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>434</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>twCards</sender>
   <signal>currentItemChanged(QTreeWidgetItem*,QTreeWidgetItem*)</signal>
   <receiver>card_editor_main</receiver>
   <slot>current_card_changed(QTreeWidgetItem*,QTreeWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFrontFace</sender>
   <signal>toggled(bool)</signal>
   <receiver>card_editor_main</receiver>
   <slot>do_frontface(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoomIn</sender>
   <signal>triggered()</signal>
   <receiver>card_editor_main</receiver>
   <slot>do_zoom()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>446</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoomReset</sender>
   <signal>triggered()</signal>
   <receiver>card_editor_main</receiver>
   <slot>do_zoom()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>446</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoomOut</sender>
   <signal>triggered()</signal>
   <receiver>card_editor_main</receiver>
   <slot>do_zoom()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>446</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>do_about()</slot>
  <slot>do_new()</slot>
  <slot>do_load()</slot>
  <slot>do_save()</slot>
  <slot>do_saveas()</slot>
  <slot>current_asset_changed(QTreeWidgetItem*,QTreeWidgetItem*)</slot>
  <slot>current_card_changed(QTreeWidgetItem*,QTreeWidgetItem*)</slot>
  <slot>do_frontface(bool)</slot>
  <slot>do_zoom()</slot>
 </slots>
</ui>
